selene/src/roblox/base.toml

```diff
@@ -67,6 +67,9 @@ type = "number"
 [[bit32.replace.args]]
 type = "number"
 
+[[bit32.replace.args]]
+type = "number"

```

Comment:
```suggestion
type = "number"
required = false
```

Reply:
as far as I could tell the first through third parameter is required:

running
```lua
bit32.replace(1, 1)
```

results in the following error:
```
21:08:29.650  bit32.replace(1, 1):1: missing argument #3 to 'replace' (number expected)  -  Edit
21:08:29.650  Stack Begin  -  Studio
21:08:29.650  Script 'bit32.replace(1, 1)', Line 1  -  Studio
21:08:29.650  Stack End  -  Studio
```

Reply:
Oh, I thought this was width.

What's the 4th parameter in your screenshot, then? Why doesn't your PR include that?

Reply:
The 4th parameter (width) is already accounted for:
![Screenshot_42](https://user-images.githubusercontent.com/55821122/150911216-37871e6d-97bd-4580-bfea-aee5b0c66fd1.png)
![Screenshot_41](https://user-images.githubusercontent.com/55821122/150911230-f521dcf5-f70d-4133-ae2a-ee1c258fca78.png)

This just seeks to add the third required variable.

Reply:
Gotcha :+1: